guides/21_Background_Filter_VideoFX_Processor.md

```diff
@@ -68,13 +68,12 @@ The following table lists the browsers and version that support background filte
 | --- | --- | 
 | Firefox | 76\+ | 
 | Chromium\-based browsers and environments, including Edge and Electron | 78\+ | 
+| Android Chrome | 110\+ | 
```

Comment:
Do we support Samsung browser?

Reply:
Will require more specific round of QA testing. Added to backlog